docs/source/faq/installation.md

```diff
@@ -0,0 +1,171 @@
+# FAQ: Installation
+
+## Why are there different versions of Manim?
```

Comment:
I would still link this somewhere more up front because i think it is also relevant for people trying to understand why something doesn't work. I mean yeah that's exactly why it's in the FAQ. But it's a pretty relevant Information in general i think 🤔

Reply:
Good point, I've added a note block to the landing page as well!

Reply:
I now just noticed there is a warning block in the installation section. 😮‍💨

Reply:
I think it is fine to have it on the landing page as well. Can't point it out too often. :-)